web/src/main/java/org/springframework/security/web/jackson2/SavedCookieMixin.java

```diff
@@ -29,24 +29,22 @@
  *
  * <pre>
  * 		ObjectMapper mapper = new ObjectMapper();
- *		mapper.registerModule(new WebServletJackson2Module());
+ * 		mapper.registerModule(new WebServletJackson2Module());
  * </pre>
  *
  * @author Jitendra Singh.
- * @since 4.2
  * @see WebServletJackson2Module
  * @see org.springframework.security.jackson2.SecurityJackson2Modules
  */
-@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY)
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS)
```

Comment:
I think we should leave this class as-is. The reason is that we want to remain backward compatible to circumstances where `comment` and `version` are still in use. This will allow applications to change when they are ready.